reports/Florida.md

```diff
@@ -37,3 +38,20 @@ A police officer grabs a protestor from behind (light grey hoodie), pull him out
 **Links**
 
 * https://twitter.com/ewcfilms/status/1268003421786906627
+
+### Police fire on protesters outside city hall | Date unknown
+
+Police open fire on protesters outside of city hall with teargas, flashbands, and apparently rubber bullets.
```

Comment:
Could you get a confirmation (e.g. using Google Streetview) of the `city hall` location claim?

Reply:
Got it
Street is 29 W. South Street
I put 4 comparison shots into this album: https://imgur.com/a/HQC9sZk

Set 1: Video begins at intersection with S. Orange Ave facing east. You can see the concrete divider in the middle of the road and the turn arrow on the left. The steeple is visible in both backgrounds.

Set 2: Video turns towards buildings south of the street. Same window and pillar design. 

Set 3. Near intersection of  29 W. South Street and Boone Ave, facing east. Building shown in both are the same, and you can see the maintenance (?) doors on the left of both pictures. 

Set 4. Facing south on Boone Ave. Video image is blurry, but crosswalk and overpass are both visible.

Reply:
Fantastic work @Ghostwaffles, I'll add that detail and then merge this one 👍